content/en/cli/pre-rendering.md

```diff
@@ -1,150 +1,146 @@
 ---
-name: CLI's Pre rendering
+name: 'Preact CLI: Pre-rendering'
 permalink: '/cli/pre-rendering'
-description: 'Preact CLI documentation for pre-rendering'
+description: 'Automatically pre-render pages using Preact CLI.'
 ---
 
-# Pre rendering pages
+# Pre-rendering Pages
 
-Preact CLI supports out of the box automatic pre-rendering of pages. By default, the homepage is always pre-rendered by executing your code in a node environment.
+Preact CLI automatically "pre-renders" your pages to static HTML so they load fast.
 
-You can give additional props and details and other URLs that you want to be pre-rendered in a js/json file as shown below.
+When building for production, Preact CLI renders your components and saves the result as static HTML. This means visitors to your site immediately see the pre-rendered HTML version, even before any JavaScript has loaded.
```

Comment:
Can we write something like this:

In production, Preact CLI renders your components....